### ISSUE TITLE
[Checkbox Group] Missing aria-checked attribute

### ISSUE THREAD

**Issue by MatthieuLebigre** [ing-bank/lion#1127]
## Expected behavior

checkbox component should have the aria-checked attribute, reflecting its state. See https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

## Actual Behavior

checkbox component doesn't have the aria-checked attribute

## Additional context

LionSwitchButton has it, so it's just adapt the same code (or probably use a mixin)

**Comment by MatthieuLebigre** [comment]
If I have time, I'll do a PR for that

**Comment by jorenbroekema** [comment]
Correct me if I'm wrong, but I am under the impression that this is not necessary.

The reason being that we actually use the native `<input type="checkbox">` in our checkbox component. As far as I understand, `aria-checked` attribute is only needed for elements that are not inherently already "checkable" elements, like in the MDN example, on `<span role="checkbox">`.

For added info, apple's VoiceOver pronounces the following when I go on `lion-checkbox` here https://lion-web-components.netlify.app/?path=/story/forms-checkbox-group--main:

```
Archimedes, unticked, Tick box
```

followed by

```
You are currently on a Tick box. To select or deselect this tick box, press control+option+space.
```

This seems correct to me

**Comment by MatthieuLebigre** [comment]
Hmm, looks like I didn't read correctly the doc :) You're right, `<input type="checkbox"` doesn't need it.
> Instead use the native HTML checkbox of `<input type="checkbox">`, which natively provides all the functionality required

I think you can close this issue then :)
